<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportToOpenIHM</class>
 <widget class="QDialog" name="ExportToOpenIHM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Project to IHM file</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Project Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lblProjectName">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>25</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Start Date:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lblStartDate">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>25</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Currency</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="lblCurrency">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>25</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QPushButton" name="cmdExport">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>41</height>
         </size>
        </property>
        <property name="text">
         <string>Export</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="cmdClose">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>41</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cmdExport</sender>
   <signal>clicked()</signal>
   <receiver>ExportToOpenIHM</receiver>
   <slot>exportProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdClose</sender>
   <signal>clicked()</signal>
   <receiver>ExportToOpenIHM</receiver>
   <slot>closeForm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>getDB()</slot>
  <slot>exportProject()</slot>
  <slot>closeForm()</slot>
 </slots>
</ui>
